config/base/params.env

```diff
@@ -1,4 +1,10 @@
 trustyaiServiceImage=quay.io/trustyai/trustyai-service:latest
 trustyaiOperatorImage=quay.io/trustyai/trustyai-service-operator:latest
 oauthProxyImage=quay.io/openshift/origin-oauth-proxy:4.14.0
-kServeServerless=disabled
\ No newline at end of file
+kServeServerless=disabled
+lmes-driver-image=quay.io/trustyai/ta-lmes-driver:latest
+lmes-pod-image=quay.io/trustyai/lm-eval-aas-flask:latest
```

Comment:
I believe you will update this to `ta-lmes-job`.

Reply:
That's right, thanks. This was writen before we decided on the image's name 👍